redshift_sqlalchemy/dialect.py

```diff
@@ -587,7 +587,10 @@ def _get_redshift_table(self, connection, table_name, schema=None, **kw):
         key = _get_relation_key(table_name, schema)
```

Comment:
The `_get_relation_key` business I copied from the postgres dialect code, but it feels kind of fragile. If somebody knows a more canonical way of parsing table and schema names so that `None` schema and `schema.table`-style table names are handled consistently, let me know.

This point is kind of tangential to this PR, but could be a source of erroneous `KeyError`s.